src/popup/components/MultisigVaultCreateReview.vue

```diff
@@ -39,6 +39,12 @@
       </DetailsItem>
 
       <DetailsItem :label="$t('multisig.consensus')">
+        <template #label>
+          <BtnHelp
+            is-custom-emit
+            @help="openModal(MODAL_CONSENSUS_INFO)"
```

Comment:
I suggest to create a custom method for this action to have a better TS support/validation.